internal/integration/fetcher.go

```diff
@@ -14,7 +14,7 @@ import (
 	"sync"
 	"time"
 
-	io_prometheus_client "github.com/prometheus/client_model/go"
+	dto "github.com/prometheus/client_model/go"
```

Comment:
I'm not really a fan of the d-word... Could we use `model` instead?

Reply:
This is how it is imported in Prometheus code itself: https://pkg.go.dev/github.com/prometheus/common/expfmt. I think we should be following that.